less/slider.less

```diff
@@ -122,11 +130,24 @@
     color: @validation-success-inverted;
   }
 
+  &__widget.show-user-answer.is-correct &__item-input-fill {
+    background-color: @validation-success;
+  }
+
   &__widget.show-user-answer.is-incorrect &__number-selection {
     background-color: @validation-error;
     color: @validation-error-inverted;
   }
 
+  &__widget.show-user-answer.is-incorrect &__item-input-fill {
+    background-color: @validation-error;
+  }
+
+  // &__widget.show-correct-answer.is-correct &__item-input-fill,
```

Comment:
Is this needed?

Reply:
Have uncommented out this code and moved it to the vanilla theme